### DISCUSSION TITLE
Is it possible to use Phoenix code on a MacPro 2019?

### DISCUSSION THREAD

**Discussion by unidia** [phcode-dev/phoenix#1788] [Q&A] [answered]
Sorry if this is a rudimentary question, but please tell me.
Is it possible to use Phoenix code on a MacPro 2019?
I downloaded the intel version and opened it, but I can't do anything but have a white canvas.

**Comment by charlypa** [comment]
@unidia 
Which version of mac os are you using?

**Reply by unidia** [reply]
Thank you for reply.
OS 11.7.9 Big Sur, Sir.

**Reply by abose** [reply, marked as answer]
Apologies @unidia . Bigsur is not supported. Please see below issue for more details.

https://github.com/phcode-dev/phoenix/issues/1565
